src/phpDocumentor/FlySystemFactory.php

```diff
@@ -12,6 +12,7 @@
 
 namespace phpDocumentor;
```

Comment:
We want to move this to the `phpDocumentor\Infrastructure` namespace since this is not part of our domain model (the interface is; because that is specific in our domain but this concrete implementation is specific to Flysystem). I could understand if the distinction is not entirely clear, please indicate if that is the case then I will try toelaborate